### PR TITLE
Enable FlaggedRevs and change group permissions in isvwiki

### PR DESCRIPTION
**Pull request by thelevking**
Group permissions:
- Stop editing of CSS/JS by anons
- Remove default patrolling from pages
- Replace it with FlaggedRevs-based settings for same groups
- Move restrictions for users

FlaggedRevs:
- Articles/Files/Templates/Categories/Modules for review
- Protection-like settings
- One status setting
- No autopromotion

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2491,6 +2492,13 @@
 			NS_HELP,
 			NS_PROJECT,
 		),
+		'isvwiki' => array(
+			NS_MAIN,
+			NS_FILE,
+			NS_TEMPLATE,
+			NS_CATEGORY,
+			828,
```
Comment (Reception123) [inline LocalSettings.php]: not sure this is possible

File: LocalWiki.php [hunk 2]
```diff
@@ -83,6 +83,31 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
 	$wgGroupPermissions['*']['createpage'] = true;
 }
 
+if ( $wgDBname === 'isvwiki' ) {
+	$wgGroupPermissions['*']['editmyusercss'] = false;
+	$wgGroupPermissions['*']['editmyuserjs'] = false;
+	$wgGroupPermissions['*']['writeapi'] = false;
+	$wgGroupPermissions['user']['editmyusercss'] = true;
```
Comment (Reception123) [inline LocalWiki.php]: True statements need to be done in wgGroupPermissions on LocalSettings

File: LocalWiki.php [hunk 3]
```diff
@@ -83,6 +83,31 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
...
+	$wgGroupPermissions['sysop']['autopatrol'] = false;
+	$wgGroupPermissions['autopatrolled']['patrol'] = false;
+	$wgGroupPermissions['confirmed']['patrol'] = false;
+	$wgGroupPermissions['sysop']['patrol'] = false;
+	
+	$wgGroupPermissions['autopatrolled']['autoreview'] = true;
+	$wgGroupPermissions['bot']['autoreview'] = true;
+	$wgGroupPermissions['confirmed']['autoreview'] = true;
+	$wgGroupPermissions['confirmed']['review'] = true;
+	$wgGroupPermissions['confirmed']['autoconfirmed'] = true;
```
Comment (Reception123) [inline LocalWiki.php]: same as above

File: LocalSettings.php (hunk 1 above)
Comment (thelevking) [inline LocalSettings.php]: NS_MODULE as an alternative? Scribunto will unreview the pages as I am aware.

Comment (Reception123) [inline LocalSettings.php]: If you think it would work with a number, than we can leave it

Comment (thelevking) [inline LocalSettings.php]: It works in https://noc.wikimedia.org/conf/highlight.php?file=flaggedrevs.php

File: LocalWiki.php (hunk 2 above)
Comment (thelevking) [inline LocalWiki.php]: Fixed (both things).

**Comment by thelevking** [comment]
`[2d6456ebdd3d1c980bac1a1a] 2017-05-21 12:22:56: Fatal exception of type "Exception"`

Don’t really know why, but there’s some errors. Module thing broke?

**Comment by Reception123** [comment]
@thelevking The error is  "FlaggedRevs given invalid tag value!"

**Comment by thelevking** [comment]
@Reception123 maybe you can just s/accuracy/status? I copied that setting from Wikimedia config and honestly didn’t know.

**Comment by Reception123** [comment]
Could be

**Pull request merged**